src/Streamly/Internal/Data/Stream/Async/Channel/Interleave.hs

```diff
@@ -198,6 +198,7 @@ getFifoSVar mrun cfg = do
             , workerThreads    = running
             , workLoop         = wloop q sv
             , enqueue          = \_ -> enqueueFIFO sv q
+            , eagerDispatch    = return ()
```

Comment:
Eager dispatch is not applicable in interleave?

Reply:
Not implemented as of now.